base/applications/utilman/dialog.c

```diff
@@ -89,26 +89,16 @@ BOOL DlgInitHandler(HWND hDlg)
     /* Refresh the list */
     ListBoxRefreshContents();
 
-    for (Globals.iArrayIndex = 0; Globals.iArrayIndex < _countof(EntriesList); Globals.iArrayIndex++)
-    {
-        if (Globals.iArrayIndex == 0)
-        {
-            /* Load the groupbox's title string */
-            LoadStringW(Globals.hInstance, IDC_GROUPBOX_OPTIONS_TITLE, EntriesList[Globals.iArrayIndex].szCaptionUtility, _countof(EntriesList[Globals.iArrayIndex].szCaptionUtility));
+    /* Load the groupbox's title string */
```

Comment:
All these LoadString calls up to (and including) `SetWindowTextW(GetDlgItem(hDlg, IDC_GROUPBOX), Globals.szFormat);` must not be done there, but only by the function that updates the items when you select a different tool from the list.